activesupport/lib/active_support/fragment_cache_key.rb

```diff
@@ -0,0 +1,9 @@
+module FragmentCacheKey
+  # Given a key (as described in ActionController::Caching::Fragments.expire_fragment),
```

Comment:
@rafaelfranca am I referencing this constant right? There's some documentation below that does it like this.